cmd/bpf2go/main.go

```diff
@@ -66,7 +66,7 @@ var targetByGoArch = map[string]target{
 	"mips64p32":   {"bpfeb", ""},
 	"ppc64":       {"bpfeb", "powerpc"},
 	"s390":        {"bpfeb", "s390"},
-	"s390x":       {"bpfeb", "s390"},
+	"s390x":       {"bpfeb", "s390x"},
```

Comment:
This is not correct since the second entry in `target` is used to define [`__TARGET_ARCH_s390`](https://github.com/libbpf/libbpf/blob/f81eef23b33c0dbf923e863a72ce51ea4d32e291/src/bpf_tracing.h#L11C34-L11C34), etc. There is no `__TARGET_ARCH_s390x` from what I can tell.

Reply:
u mean the define in bpf_tracing.h right I think the same target `_s390` seems to be used for both arch 32bits and 64bits at least that is what I saw in https://github.com/iovisor/bcc/commit/7d7b6e888a1fe0af1db037bed2300c16cf32994c

Reply:
u are right reworked the diffs PTAL